pandas/core/arrays/boolean.py

```diff
@@ -557,6 +557,30 @@ def _values_for_argsort(self) -> np.ndarray:
         data[self._mask] = -1
         return data
 
+    def any(self, skipna=True):
```

Comment:
What happens with `np.any` with this? Do we need any keywords for compatibility?

Is the expected behavior here different from `nanops.nanany`? / `nanops.nanall`?